demo/md-styles.html

```diff
@@ -0,0 +1,787 @@
+<link rel="import" href="../polymer/polymer-element.html">
```

Comment:
If we do decide to go the 2nd route, it would be good to split that effort out from the documentation of the event and how to use it.